plugin.video.manoto/default.py

```diff
@@ -1,115 +1,10 @@
-import urllib,urllib2,re,os,cookielib,string
-import xbmcplugin,xbmcgui,xbmcaddon
-from BeautifulSoup import BeautifulSoup
-
-addon = xbmcaddon.Addon('plugin.video.manoto')
-profile = xbmc.translatePath(addon.getAddonInfo('profile'))
-
+import xbmc, xbmcgui, os, xbmcaddon
 
 __settings__ = xbmcaddon.Addon(id='plugin.video.manoto')
-__language__ = __settings__.getLocalizedString
-
 home = __settings__.getAddonInfo('path')
 icon = xbmc.translatePath(os.path.join(home, 'icon.png'))
+li = xbmcgui.ListItem ('manoto')
+li.setThumbnailImage(icon)
```

Comment:
`setThumbnailImage` is deprecated, please use `setArt` instead. See: https://codedocs.xyz/xbmc/xbmc/group__python__xbmcgui__listitem.html